test/specs/rules/disallow-space-after-keywords.js

```diff
@@ -1,7 +1,7 @@
 var Checker = require('../../../lib/checker');
 var assert = require('assert');
 
-describe('rules/disallow-space-after-keywords', function() {
+describe.only('rules/disallow-space-after-keywords', function() {
```

Comment:
Lol so I thought of that too since it's annoying but forgot that we could
just use the rule!

On Tuesday, May 12, 2015, Mike Sherov <notifications@github.com> wrote:

> In test/specs/rules/disallow-space-after-keywords.js
> <https://github.com/jscs-dev/node-jscs/pull/1364#discussion_r30147217>:
>
> > @@ -1,7 +1,7 @@
> >  var Checker = require('../../../lib/checker');
> >  var assert = require('assert');
> >
> > -describe('rules/disallow-space-after-keywords', function() {
> > +describe.only('rules/disallow-space-after-keywords', function() {
>
> By the way, perhaps we should disallow this identifier name... If only we
> knew of a linter that provided this functionality.... ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/node-jscs/pull/1364/files#r30147217>.
>


-- 
-Henry Zhu
Georgia Institute of Technology
Industrial Engineering '14
Computer Science Minor